src/main/external-resources/renderers/FetchTV.conf

```diff
@@ -0,0 +1,39 @@
+#----------------------------------------------------------------------------
+# Profile for Fetch TV (Mighty and Mini) @ https://www.fetchtv.com.au/
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Fetch TV
+RendererIcon = fetch-tv.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Linux/3.14.28-1.6, UPnP/1.0, DLNADOC/1.50
+# ============================================================================
+#
+
+UserAgentSearch = Linux/3.14.28-1.6.*DLNADOC/1.50
+
+TranscodeVideo = MPEGTS-H264-AC3
+TranscodeAudio = MP3
+MuxLPCMToMpeg = false
```

Comment:
Thank you for your contribution. There's a couple of things that's hard for an outsider to know. ```MuxLPCMToMpeg``` isn't used if ```MediaInfo = true```, but the ```Supported``` lines you've defined isn't used unless ```MediaInfo = true```.

Parsing without MediaInfo is the old, crude way of parsing with UMS, and in most cases it won't be very accurate. You should enable MediaInfo, so I'd replace this line with ```MediaInfo = true```.